src/components/Navbar/index.tsx

```
@@ -384,16 +393,20 @@ const Navbar = () => {
-                <MobileNavItemLabel>{item.label}</MobileNavItemLabel>
-              </MobileNavItem>
-            ))}
+            ].map((item) => {
+              const Item = item.icon;
+              return (
+                <MobileNavItem
+                  active={activePath.pathname == item.location}
+                  onClick={() => {
+                    navigate(item.location);
+                  }}
+                >
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)